monitoring/install-ks-monitoring.sh

```diff
@@ -120,29 +163,45 @@ if [ $opt == "core" ];then
 
     wait-for-object $ctx $ns deployment grafana
 
+    if [ $env == "ocp" ];then
+         kubectl -n $ns apply -f ${SCRIPT_DIR}/grafana/grafana-route.yaml
+    fi 
+
     # Set the Thanos URL for Prometheus remote write
     export thanos_host="thanos-receive.ks-monitoring.svc.cluster.local:19291"
-    sed -e s/%THANOS_HOST%/$thanos_host/g ${SCRIPT_DIR}/prometheus/prometheus-custom-values.yaml >& /tmp/prometheus-custom-values-set.yaml
+    endpoint="http://$thanos_host"
+    sed -e s,%THANOS_HOST%,$endpoint,g -e s,%ENABLE_KUBEAPISERVER%,true,g ${SCRIPT_DIR}/prometheus/prometheus-custom-values.yaml >& /tmp/prometheus-custom-values-set.yaml
```

Comment:
(A) See https://www.gnu.org/software/bash/manual/bash.html#Redirecting-Standard-Output-and-Standard-Error for the preferred syntax for redirecting both stdout and stderr.

(B) Why redirect stderr? You do not want error messages in that output YAML file, right?

Reply:
Good catch! I update the script to redirect the stdout only.